sample-app/src/hooks/useAutocomplete.tsx

```diff
@@ -0,0 +1,20 @@
+import { useRef, useState } from "react";
+import { AutocompleteResult, useAnswersActions } from '@yext/answers-headless-react';
+
+export function useAutocomplete(isVertical: boolean): [AutocompleteResult[], () => Promise<void>] {
+  const answersActions = useAnswersActions();
+  const autocompleteNetworkIds = useRef({ latestRequest: 0, responseInState: 0 });
+  const [ autocompleteResults, setAutoCompleteResults ] = useState<AutocompleteResult[]>([]);
+  async function executeAutocomplete () {
+    const requestId = ++autocompleteNetworkIds.current.latestRequest;
+    let response = null;
+    isVertical
```

Comment:
is it possible to do this with a `const response`?

Reply:
oh yeah